### PR TITLE
[#2056] Remove save button from project settings

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by k88hudson** [comment]
@gvn R?

File: www_src/pages/project-settings/project-settings.jsx [hunk 1]
```diff
@@ -2,79 +2,103 @@ var React = require('react/addons');
...
+
   componentWillMount: function () {
-    var _this = this;
 
-    // Build up URI for API requests
-    var uri = '/users/' + this.state.params.user + '/projects/' + this.state.params.project;
+    this.load();
+
+    // Set up the save method when we press the back button
+    this.props.update({
```
Comment (k88hudson) [inline www_src/pages/project-settings/project-settings.jsx]: This overrides the back  button method on the root component `to call this.save`

File: www_src/pages/project-settings/project-settings.jsx [hunk 2]
```diff
@@ -2,79 +2,103 @@ var React = require('react/addons');
...
+      onBackPressed: () => {
+        this.save(() => window.Android.goBack());
+      }
+    });
+
+  },
+
+  componentDidUpdate: function (prevProps) {
+    if (this.props.isVisible && !prevProps.isVisible) {
```
Comment (k88hudson) [inline www_src/pages/project-settings/project-settings.jsx]: if the page changes from `paused` to `resumed` via Android events, reload data

**Comment by gvn** [comment]
R+!

**Pull request merged**